tests/unit/test_charm.py

```diff
@@ -5,6 +5,8 @@
 
 """Charm unit tests."""
 
+# pylint: disable=protected-access
```

Comment:
I don't quite see private module acess, would this still be required?

Reply:
I think this is just caused by me copying the file header from somewhere else, removed, thanks!